CMakeLists.txt

```diff
@@ -289,6 +289,7 @@ if(BUILD_COMMON_LWS)
       set_target_properties(kvsCommonLws PROPERTIES VERSION ${KINESIS_VIDEO_PRODUCER_C_VERSION} SOVERSION ${KINESIS_VIDEO_PRODUCER_C_MAJOR_VERSION})
     endif()
     target_link_libraries(kvsCommonLws
+            ${PRODUCER_CRYPTO_LIBRARY}
             ${OPENSSL_CRYPTO_LIBRARY}
```

Comment:
Just want to leave a note that the existing `${OPENSSL_CRYPTO_LIBRARY}` and `${OPENSSL_SSL_LIBRARY}` should likely not be here since `${PRODUCER_CRYPTO_LIBRARY}` will cover both openssl and mbedtls cases. This can be changed and tested in a separate PR to /develop.